ocpmodels/models/dimenet.py

```diff
@@ -75,6 +76,14 @@ def forward(self, data):
             edge_index, num_nodes=x.size(0)
         )
 
+        # Cap no. of triplets during training.
+        if self.training:
```

Comment:
where is this defined?

Reply:
This gets set to `True` when we call `model.train()`:
- https://github.com/Open-Catalyst-Project/baselines/blob/master/ocpmodels/trainers/energy_trainer.py#L148
- https://github.com/Open-Catalyst-Project/baselines/blob/master/ocpmodels/trainers/forces_trainer.py#L276

And to `False` when we call `model.eval()`.

Reply:
Ah neat didn't know, looks good!